module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,24 @@
+#include <algorithm>
+#include <forward_list>
+#include <string>
+
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& list){
+    list.sort();
```

Comment:
Just a tip - please notice, that you can inject you own comparator to sort :) https://en.cppreference.com/w/cpp/container/list/sort  <--- pls look at the section parameters :)

Reply:
Great tip, this and lambda helped me to simplify the code. Thanks, I haven't thought about this at the beginning.

Reply:
But it's not sorted in a proper way now. Our tests suck and they have not detected it. Try it on an example from the presentation - "One", "Two", "Three", etc.